unison-src/transcripts/api-find.output.md

```diff
@@ -60,9 +60,9 @@ GET /api/find?query=http
             "contents": {
                 "bestFoundTermName": "y",
                 "namedTerm": {
-                    "termHash": "#emomp74i93h6ps0b5sukke0tci0ooba3f9jk21qm919a7act9u7asani84c0mqbdk4lcjrdvr9olpedp23p6df78r4trqlg0cciadc8",
+                    "termHash": "#emomp74i93",
```

Comment:
We're currently inconsistent about whether we use short or long hashes throughout. It's tricky, because we produce shorthashes low down within UCM before we know where the hash will be used.